codeserver/ubi9-python-3.11/Dockerfile.cpu

```diff
@@ -1,7 +1,35 @@
-ARG BASE_IMAGE
-FROM ${BASE_IMAGE}
+####################
+# base             #
+####################
+FROM registry.access.redhat.com/ubi9/python-311:latest AS base
```

Comment:
Since we're moving to konflux soon anyway, what do you think about moving from `latest` to an actual sha/tag for the base images? This may help also #901 maybe?

@jiridanek WDYT?

Reply:
absolutely not in this PR

I'm guessing if we let renovate to run, it will rewrite the tags first thing ;p so maybe no manual effort needed

good thought, tho

Reply:
* https://github.com/opendatahub-io/notebooks/issues/928